package exceptions;

public class TestTask4 {
    public static void main(String[] args) {
       TaskException.makeUser("Evgeniy—Ä", "Passwd1", "passwd1");
    }
}
